### DISCUSSION TITLE
Design constraints of mission concept

### DISCUSSION THREAD

**Discussion by capsulecorplab** [ODU-CGA-CubeSat/sealion-mission-architecture#19] [Ideas]
As per https://github.com/ODU-CGA-CubeSat/flight-software/discussions/18#discussioncomment-884492, this discussion is to capture design constraints of the mission concept

**Comment by paclark5** [comment]
Per Dr. Popescu, communication protocol is expected to be AX.25, amateur data link layer, but he was not sure if UDP might be used. Will confirm with Dr. Asundi.

**Reply by capsulecorplab** [reply]
Should verify if spacecraft can only send or receive data

**Comment by capsulecorplab** [comment]
Currently only have 4 booms and will be using two accelerometers, as per discussion w/ @BhagatjiJimesh during 6/30 architecture meeting

**Reply by capsulecorplab** [reply]
OBC may provide absolute value for acceleration or frequency. Underlying assumption is that IMU will have accelerometer. as per discussion w/ @BhagatjiJimesh during 6/30 architecture meeting
